modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/computation/BulkScrollerComputation.java

```diff
@@ -284,17 +286,38 @@ protected void updateStatusAsScrolling(ComputationContext context, String comman
                 BulkCodecs.getStatusCodec().encode(delta));
     }
 
+    /**
+     * @deprecated since 11.5, use {@link #updateStatusAfterScroll(ComputationContext, String, String, int)} instead
+     **/
+    @Deprecated(since = "11.5", forRemoval = true)
     protected void updateStatusAfterScroll(ComputationContext context, String commandId, String errorMessage) {
-        updateStatusAfterScroll(context, commandId, 0, errorMessage, false);
+        int errorCode = StringUtils.isEmpty(errorMessage) ? 0 : HttpStatus.SC_INTERNAL_SERVER_ERROR;
+        updateStatusAfterScroll(context, commandId, errorMessage, errorCode);
+    }
```

Comment:
I think deprecation is not needed there, such APIs were already modified when we introduce the `limited` parameter.